com.unity.multiplayer.mlapi/Runtime/Serialization/Pooled/BitStreamPool.cs

```diff
@@ -9,10 +9,14 @@ namespace MLAPI.Serialization.Pooled
     /// </summary>
     public static class BitStreamPool
     {
-        private static byte createdStreams = 0;
+        private static uint createdStreams = 0;
         private static readonly Queue<WeakReference> overflowStreams = new Queue<WeakReference>();
         private static readonly Queue<PooledBitStream> streams = new Queue<PooledBitStream>();
 
+        private const uint MaxBitPoolStreams = 1024;
+        private const uint MaxCreatedDelta = 768;
```

Comment:
I think changes for this file and also other BitStream files should be under a separate PR that suggests improvements/refactor/enhancements over what we have today — and we could discuss other possibilities over there IMHO.

Reply:
This is needed.  When we talk later today on the RPCReceiveParams, I can explain.  This was something taken out of a previous tree that is required if you want to have more than 256 objects moving around.